ui-components/src/navigation/Tabs.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react"
+import { Tab, Tabs, TabList, TabPanel } from "./Tabs"
+
+export default {
+  title: "Navigation/Tabs",
+  decorators: [(storyFn: any) => <div style={{ padding: "1rem" }}>{storyFn()}</div>],
+}
+
+export const Default = () => {
+  return (
+    <Tabs>
+      <TabList>
+        <Tab>Title 1</Tab>
+        <Tab>Title 2</Tab>
+        <Tab>Long Tab Title 3</Tab>
+        <Tab disabled>Disabled Tab</Tab>
+      </TabList>
+
+      <TabPanel>
+        <h2>Any content 1</h2>
+        <p>Weee</p>
```

Comment:
weeeeeee

Reply:
Haha, I forgot I'd left those in there 😅